lib/ansible/modules/network/ios/_ios_template.py

```diff
@@ -147,9 +147,7 @@ def main():
     candidate = NetworkConfig(contents=module.params['src'], indent=1)
 
     result = {'changed': False}
-    warnings = list()
-    check_args(module, warnings)
-    result['warnings'] = warnings
+    check_args(module)
```

Comment:
i cannot imagine people look at deprecated files first for examples ...., but they do see changes to them and then try to use those to justify subsequent changes.